lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runtime/azbatch/util/batch/AzureBatchHelper.java

```diff
@@ -139,7 +156,10 @@ public void submitTask(final String jobId, final String taskId, final URI jobJar
     final TaskAddParameter taskAddParameter = new TaskAddParameter()
         .withId(taskId)
         .withResourceFiles(resources)
-        .withCommandLine(command);
+        .withContainerSettings(createTaskContainerSettings())
+        .withCommandLine(command)
+        .withUserIdentity(
+            new UserIdentity().withAutoUser(new AutoUserSpecification().withElevationLevel(ElevationLevel.ADMIN)));
```

Comment:
We only want to elevate the level when necessary. Can we limit this to containers only.